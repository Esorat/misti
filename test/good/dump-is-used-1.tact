fun test() {
    dump(42); // Should be highlighted
}
