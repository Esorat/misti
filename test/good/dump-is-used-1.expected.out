[INFO] DumpIsUsed: Found `dump` usage
test/good/dump-is-used-1.tact:2:5:
  1 | fun test() {
> 2 |     dump(42); // Should be highlighted
          ^
  3 | }
Help: Using `dump` in production code can sometimes indicate complex code that requires additional review
See: https://nowarp.io/tools/misti/docs/detectors/DumpIsUsed